src/components/tooltip/tooltip.test.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import user from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+import Tooltip from './tooltip.jsx';
+
+describe('tooltip', () => {
+  const requiredProps = {
+    children: <div id="my-component" aria-describedby="my-component-tooltip">Child</div>,
+    id: 'my-component-tooltip',
+    text: 'Help Text',
+  };
+
+  it('has defaults', () => {
+    render(<Tooltip {...requiredProps} />);
+    expect(screen.getByText('Child').parentElement).toMatchSnapshot();
+
+    user.hover(screen.getByText('Child'));
```

Comment:
I might also start naming the user-event as just `user`. This sounds way slicker. user.click, user.hover, user.keyboard (eh?), user.type

Reply:
Yeah! I saw that kent dodds did it in one of his examples and I really liked the way it makes the test read